Modules/posixmodule.c

```diff
@@ -312,7 +313,7 @@ corresponding Unix manual entries for more information on calls.");
 #  include <sys/syscall.h>
 #endif
 
-#if defined(MS_WINDOWS)
+#if defined(MS_WINDOWS) && !defined(MS_WINDOWS_NON_DESKTOP)
```

Comment:
This should be `!defined(MS_WINDOWS_GAMES)`. 

`GetConsoleScreenBufferInfo()` is available for `WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_APP | WINAPI_PARTITION_SYSTEM)`.